Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Messagebox/Enumerations/Enumerations.cs

```diff
@@ -60,7 +60,8 @@ public enum ExtendedMessageBoxCustomButtonOptions
         /// <summary>Use a two button layout.</summary>
         TWOBUTTONS = 1,
         /// <summary>Use a three button layout.</summary>
-        THREEBUTTONS = 3
+        THREEBUTTONS = 3,
+        NONE = 4
```

Comment:
None (Or unknown) should be 0 by default, so that it does not need to be initialised to a specific value.

Reply:
Will resolve today.

Reply:
@Smurf-IV It's now fixed by defaulting `NONE` to 0, and OK